tool/update_perfetto_assets.dart

```diff
@@ -0,0 +1,73 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+void main(List<String> args) {
+  final mainDevToolsDirectory = Directory.current;
+  if (!mainDevToolsDirectory.path.endsWith('/devtools')) {
+    throw Exception(
+      'Please execute this script from your top level '
+      '\'devtools/\' directory. Running the `update_perfetto.sh` script will'
+      'automatically do this.',
+    );
+  }
+
+  final perfettoDistDir = Directory.fromUri(
+    Uri.parse(
+        '${mainDevToolsDirectory.path}/third_party/packages/perfetto_compiled/lib/dist'),
+  );
+
+  // Find the new perfetto version number.
+  String newVersionNumber = '';
+  final versionRegExp = RegExp(r'v[0-9]+[.][0-9]+-[0-9a-f]+');
+  final entities = perfettoDistDir.listSync();
+  for (FileSystemEntity entity in entities) {
```

Comment:
Something about this feels like it may cause issues in the future. Is there a specific file we could focus on that would for sure have the version?

Reply:
It's a single folder:
![Screen Shot 2022-10-06 at 11 28 11 AM](https://user-images.githubusercontent.com/43759233/194390781-bc5288ec-e84d-4633-927a-b2d59c801f62.png)

Reply:
ahhhh it is a filename! Sounds good 😄